srcpkgs/xdgmenumaker/template

```diff
@@ -12,10 +13,9 @@ distfiles="https://github.com/gapan/${pkgname}/archive/${version}.tar.gz"
 checksum=a8319db5998ea1c49e52f6d04aad40334daea8047840c6762b4e7f02082b0573
 patch_args="-Np1"
 
-do_build() {
-	make
-}
-
-do_install() {
-	make DESTDIR="${DESTDIR}" install
+pre_install() {
+	# Use python3.x
+	vsed -i 's;#!/usr/bin/env\spython;#!/usr/bin/python3;' src/xdgmenumaker
```

Comment:
I think this can also be achieved by setting the `python_version` variable in the template, no need to use vsed.

Reply:
`python_version` works, thank you!